ring/token_range_test.go

```diff
@@ -195,6 +200,12 @@ func TestCheckingOfKeyOwnership(t *testing.T) {
 
 	// Generate ring
 	ringDesc := &Desc{Ingesters: generateRingInstances(gen, instancesPerZone*numZones, numZones, numTokens)}
+	// Switch states of some ingesters
```

Comment:
I have a feedback similar to Bryan's one. The comment says "some ingesters" but I read the code as "all ingesters".